src/python/WMCore/Services/Requests.py

```diff
@@ -236,7 +227,7 @@ def f():
 
         headers["Content-length"] = str(len(encoded_data))
 
-        assert isinstance(encoded_data, type('string')), \
+        assert isinstance(encoded_data, str), \
```

Comment:
From the previous check, I assumed we'd be interested only in 8-bit strings, not in unicode strings. Ok, I'll update the code soon